### ISSUE TITLE
Websocket creation event failed and did not trigger the 'error' addeventlistener and bun crashed.

### ISSUE THREAD

**Issue by csrk18** [oven-sh/bun#11706]
### How can we reproduce the crash?

Try to open a new websocket and it  returns an error. The error addEventListener is not activated and bun crashes.  

### Relevant log output

```shell
panic(main thread): attempt to use null value
```

### Stack Trace (bun.report)

Bun v1.1.12 ([`43f0913`](<https://github.com/oven-sh/bun/tree/43f0913c3875cbf260c60868268adc5953b09a80>)) on windows x86_64 [AutoCommand]

**panic**: attempt to use null value

- [`writer.zig:17`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/src/deps/zig/lib/std/io/writer.zig#L17>): `write`
- [`ConsoleObject.zig:798`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/src/bun.js/ConsoleObject.zig#L798>): `format2`
- `??`
- [`StringImpl.h:1157`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/D:/a/WebKit/WebKit/WebKitBuild/WTF/Headers/wtf/text/StringImpl.h#L1157>): `WTF::StringImpl::deref`
- *2 unknown/js code*
- [`atomic:979`](<https://github.com/oven-sh/bun/blob/43f0913c3875cbf260c60868268adc5953b09a80/C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.39.33519/include/atomic#L979>): `std::_Atomic_storage<unsigned int,4>::store`
- *3 unknown/js code*

**Comment by Jarred-Sumner** [comment]
Assuming that this also involves a `Worker`, fixed in Bun v1.1.13

**Comment by csrk18** [comment]
Thanks for the quick response. I just did some more checks on why this is happening. I had just the ws.addEventlistener("error") in my code which was not getting triggered. When I added the ws.onerror code it triggered but still crashed. Then I had to catch the 'undefined' error in code and shutdown the process manually instead of bun crashing.  If the server on the other side is down, websocket request crashes bun. If this is what you have seen already and fixed, I will await the 1.1.13. All the best!
